src/app/containers/MostRead/Canonical/index.jsx

```diff
@@ -18,6 +19,55 @@ import {
   MOST_READ_CLIENT_REQUEST,
   MOST_READ_FETCH_ERROR,
 } from '#lib/logger.const';
+import useViewTracker from '#hooks/useViewTracker';
+import useClickTrackerHandler from '#hooks/useClickTrackerHandler';
+
+const TrackedMostReadLink = ({
```

Comment:
A bit of a decision here maybe, this component exists solely to add tracking support to the Most Read component, if we had migrated Most Read into Simorgh you would notionally be adding just a few lines of code to the existing Most Read component rather than redeclaring all the props etc

Reply:
Thanks @andrewscfc
Yeah this would certainly be simpler if the most read component lived in Simorgh, but of all the components that have been migrated to Simorgh from Psammead I feel like this would probably be the least specific to the BBC. Perhaps this is something we could bring up in the tech meeting?

Reply:
That's a fair point @HarryVerhoef, I think trouble I have with psammead is most our components could not be used outside of a BBC context without some changes, see the props for most read:
<img width="953" alt="image" src="https://user-images.githubusercontent.com/9645462/121176970-c140f580-c854-11eb-85c4-01e7a442e648.png">
You choose a `service` that influences it's style, RTL language considerations stuff like that. If it was a truely generic component you might choose; language, RTL that sort of thing. 

You are right though it is one of our more generic components that could fairly easily be modified to be reusable outside of the BBC. 

Feel free to take the PR forward as is from my side 🙂

Reply:
Good point @andrewscfc, the props are very specific to Simorgh. Happy to migrate this from Psammead 👍